src/benchmarks/micro/libraries/System.Numerics.Vectors/Perf_Vector2.cs

```diff
@@ -11,55 +11,127 @@ namespace System.Numerics.Tests
     [BenchmarkCategory(Categories.Libraries, Categories.SIMD, Categories.JIT)]
     public class Perf_Vector2
     {
+        [Benchmark]
+        public Vector2 CreateFromScalar() => new Vector2(1.0f);
+
+        [Benchmark]
+        public Vector2 CreateFromScalarXYBenchmark() => new Vector2(1.0f, 2.0f);
+
+        [Benchmark]
+        public Vector2 OneBenchmark() => Vector2.One;
+
+        [Benchmark]
+        public Vector2 UnitXBenchmark() => Vector2.UnitX;
+
+        [Benchmark]
+        public Vector2 UnitYBenchmark() => Vector2.UnitY;
+
+        [Benchmark]
+        public Vector2 ZeroBenchmark() => Vector2.Zero;
```

Comment:
I am afraid that the benchmarks that just return a field would report `0ns` since there are low chances of regressing them (due to our detailed code reviews in runtime repo) I think it would be better to not add them

Reply:
Same as with `Identity`, this is a special intrinsic property.